### ISSUE TITLE
Number fields: add narrow class groups

### ISSUE THREAD

**Issue by AurelPage** [LMFDB/lmfdb#994]
From /roadmap: Data on narrow class numbers.

**Comment by jwj61** [comment]
Data for most fields is ready to add to the database.  To decide exactly where/how to add it, I need to know how much do we want on narrow class groups.  

Do we want to search on them, and if so, on both narrow class numbers and narrow class groups?

**Comment by roed314** [comment]
I think yes: we should add `narrow_class_group` and `narrow_class_number` to `nf_fields`.

**Comment by jwj61** [comment]
So searching on both (or else e.g. narrow_class_group would go in the extras table).

**Comment by roed314** [comment]
Yeah, I think it makes sense to add both to the search options for number fields (and probably start using [advanced search](https://github.com/LMFDB/lmfdb/issues/5286) at the same time).

**Comment by roed314** [comment]
The reason to separate the extra table is for storage space reasons (which makes sequential scans slow).  What percentage increase would adding these to `nf_fields` involve?  It's currently [181 GB](https://beta.lmfdb.org/api/stats)

**Comment by roed314** [comment]
Currently the data in `nf_fields` uses 26GB and the data in `nf_fields_extras` uses 164GB.

```python
sage: db.table_sizes()["nf_fields"]
{'nrows': 22175431, 'index_bytes': 39509917696, 'toast_bytes': 7151616, 'table_bytes': 26065502208, 'total_bytes': 229825503232, 'extras_bytes': 164239966208, 'ncounts': 12699, 'counts_bytes': 2908160, 'nstats': 120, 'stats_bytes': 57344}
```

**Comment by jwj61** [comment]
The narrow class groups for almost 22 million fields takes 428 MB in a plain text file.  I am guessing that means it goes into the regular table.

**Comment by roed314** [comment]
Yeah, I think it's worth it.

**Comment by jwj61** [comment]
I had a failed attempt at adding data via a rewrite (lost connection).  When I tried again, it failed because nf_fields is locked, probably from the first attempt.  Can you unlock it?

**Comment by roed314** [comment]
I don't see a lock now; can you try again?

**Comment by jwj61** [comment]
Great.  It will be several hours before it gets to the stage where it is relevant.

**Comment by jwj61** [comment]
It finially finished (in the past hour).  I will leave it for now so the front end work can be done at the workshop.
